kits/melon/package.json

```diff
@@ -31,7 +33,9 @@
     "eth-ens-namehash": "^2.0.8"
   },
   "devDependencies": {
+    "@aragon/kits-beta-base": "^1.0.0",
```

Comment:
I think this should be a dependency (not a dev dependency) since the entire kit is based off of it, right?

Reply:
It's actually not needed. I think it was meant to be used for deploying deps, but there's custom script here (with Agent app). So removing it.